dashboard/app/controllers/courses_controller.rb

```diff
@@ -1,6 +1,14 @@
 class CoursesController < ApplicationController
   def index
-    course_name = params[:course].tr('-', '_').titleize
+    @recent_courses = current_user.try(:recent_courses)
+    @is_teacher = !!(current_user && current_user.teacher?)
```

Comment:
agreed. i was going to change this in a future PR, but wanted this one to mostly just be moving stuff